From 165c0f787e39b5734816a37c359c8cd816c007b1 Mon Sep 17 00:00:00 2001
From: Ong Hean Loong <hean.loong.ong@intel.com>
Date: Mon, 19 Dec 2016 17:03:35 +0800
Subject: [PATCH 064/151] FogBugz #421702-1: Device tree bindings Frame Buffer
 II for Display Port IP

commit  6112f6517c779e58be176b56ac6a2e957f94c9b6 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Device tree bindings required for the Intel PSG (Altera) Frame Buffer II.

Signed-off-by: Ong Hean Loong <hean.loong.ong@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 .../bindings/video/altr,vip-fb2.txt           | 22 +++++++++++++++++++
 1 file changed, 22 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/video/altr,vip-fb2.txt

diff --git a/Documentation/devicetree/bindings/video/altr,vip-fb2.txt b/Documentation/devicetree/bindings/video/altr,vip-fb2.txt
new file mode 100644
index 000000000000..296988aec88d
--- /dev/null
+++ b/Documentation/devicetree/bindings/video/altr,vip-fb2.txt
@@ -0,0 +1,22 @@
+Altera Video and Image Processing(VIP) Frame Buffer II bindings
+
+Required properties:
+- compatible: "altr,vip-frame-buffer-2.0"
+- reg: Physical base address and length of the framebuffer controller's
+  registers.
+- max-width: The width of the framebuffer in pixels.
+- max-height: The height of the framebuffer in pixels.
+- bits-per-symbol: only "8" is currently supported
+- mem-port-width = the bus width of the avalon master port on the frame reader
+
+Example:
+
+dp_0_frame_buf: vip@0x100000280 {
+	compatible = "altr,vip-frame-buffer-2.0";
+	reg = <0x00000001 0x00000280 0x00000040>;
+	altr,max-width = <1920>;
+	altr,max-height = <1080>;
+	altr,bits-per-symbol = <8>;
+	altr,mem-port-width = <128>;
+};
+
-- 
2.26.1

